CHANGELOG.md

```diff
@@ -3,6 +3,43 @@ aws-parallelcluster-cookbook CHANGELOG
 
 This file is used to list changes made in each version of the AWS ParallelCluster cookbook.
 
+2.4.1
+-----
+
+**ENHANCEMENTS**
+- Install IntelMPI on Alinux, Centos 7 and Ubuntu 1604
+- Upgrade EFA to version 1.4.1
+
+**CHANGES**
+- Torque: upgrade to version 6.1.2
+- Run all node daemons and cookbook recipes in isolated Python virtualenvs. This allows our code to always run with the
```

Comment:
I'd mention this as an enhancement since it should be a kind of change that doesn't impact customers while giving a lot of value

Reply:
makes sense